tests/sanity.cpp

```diff
@@ -1547,6 +1657,7 @@ int main(int argc, char *argv[])
     tcase_add_test(tc_basic, basic);
     tcase_add_test(tc_basic, basic_with_tokens);
     tcase_add_test(tc_basic, basic_unaligned_mapping);
+    tcase_add_test(tc_basic, child_thread_pins_buffer);
```

Comment:
Would be good if you use the same naming scheme -- i.e., basic_*

Reply:
good idea, done.